docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.rst

```diff
@@ -35,3 +35,190 @@ Iron Deficiency Risk Effects
+
+Alternative Approach
+^^^^^^^^^^^^^^^^^^^^^^
+
+Because the relative risks for maternal mortality are defined in terms of hemoglobin concentration rather than iron deficiency specifically, if the effect size of an intervention is measured in terms of hemoglobin concentration, the relative risks may be applied directly to the maternal disorders burden rather than through the typical approach of calculating the "risk-deleted" burden and then multiplying by the mean relative risk. This alternative approach will avoid confusion regarding the differing definitions of the iron deficiency risk exposure, TMREL, and PAF for maternal disorders.
+
+Chris Murray and Theo Vos suggested that this approach be used for estimating the impact of interventions that act on hemoglobin concentration on maternal disorders.
+
+Notably, this alternative approach was found to estimate similar (but slightly greater) intervention impact to the standard approach using the location-specific TMREL exposure estimation approach specific and exposure parameters specific to the pregnant and lactating population.
+
+.. todo::
+
+  Cite the maternal anemia project repo here.
+
+Dietary Iron Deficiency
++++++++++++++++++++++++
+
+The dietary iron deficiency cause in GBD 2019 has a population attributable fraction of 100% with the iron deficiency risk factor. This risk-outcome relationship is not described in full in this document page.
+
+When choosing to model dietary iron deficiency, consideration should be paid to the other causes of iron deficiency (iron responsive anemias), described on the :ref:`Anemia Impairment Documentation Page <2019_anemia_impairment>`.
+
+Vivarium Modeling Strategy
+--------------------------
+
+Maternal Disorders
+++++++++++++++++++++
+
+The recommended approach for modeling the relationship between hemoglobin and maternal disorders is described in the `Alternative Approach`_ section in this document. Therefore, there is no need to use the PAF in order to apply the relative risks.
+
+Instead, the relative risks should be applied to both the YLD (or incidence) and YLL (CSMR or EMR) rate for maternal disorders (or a specific modeled individual maternal disorder), such that:
+
+.. math:: 
+
+    rate_i = rate_\text{GBD} * 1 / e^{\text{ln(RR)} * (Hgb_i - Hgb_\text{GBD})}
+
+Where, 
+
+.. list-table:: Parameter Definitions
+   :widths: 5 5 5
+   :header-rows: 1
+
+   * - Parameter
+     - Definition
+     - Note
+   * - :math:`Hgb_i`
+     - Hemoglobin value for an individual pregnant simulant in g/dL
+     - Needs unit conversion! (Exposure typically defined in g/L)
```

Comment:
Should we just go ahead and be more explicit about this? E.g. "Hgb value pulled from GBD are in g/L, so divide them by 10."